public_html/css/main.scss

```diff
@@ -1400,6 +1413,36 @@ a.list-group-item {
   }
 }
 
+.extraField_dynamicContainer {
+  width:100%;
+  float:left;
+  margin:10px 0;
+  select { width:90%; float:left; }
+  button {
+    width: 10%;
+    border: 0;
+    float: left;
+    border-radius: 5px;
+    padding: 7px;
+    height: 34px;
+    @extend .btn-danger;
+  }
+}
+
+.extraFieldsBtn {
+    -webkit-appearance: none;
+    @extend .btn-default;
+    background: $blue;
+    width:100%;
+    float:left;
+    margin:10px 0;
+    border: 0;
+    color: #fff;
+    padding: 10px;
+    border-radius: 5px;
+}
+
+
```

Comment:
I can't find these classes surrounding the public and oai check boxes. This looks like it pertains to changes for the select box. If so, revert.